components/heatmap/HeatMap.vue

```diff
@@ -249,10 +249,11 @@ export default {
         total += parseFloat(
           this.well_values[this.display_option].data[well_idx]
             .slice(-MIN_NUM_DATAPOINTS_FOR_MEAN)
-            .reduce((a, b) => a + b, 0)
+            .reduce((a, b) => a + b, 0) /
```

Comment:
`this.well_values[this.display_option].data[well_idx].slice(-MIN_NUM_DATAPOINTS_FOR_MEAN)` is duplicated which creates a lot of visual noise and makes this hard to read, it should be pulled out into its own variable